driveteam.py

```diff
@@ -0,0 +1,59 @@
+import wpilib
+
+class DriveTeam:
+
+    pilot = wpilib.XboxController(0)
+
+    copilot = wpilib.XboxController(1)
```

Comment:
When working with classes it's good to consider adding an `__init__ `method (constructor) where you can initialize any required variables. 

It would look like this:

```
   def __init__(self):
        self.pilot = wpilib.XboxController(0)
        self.copilot = wpilib.XboxController(1)
```

You can read more on class constructors in python [here](https://pythonbasics.org/constructor/)